doc/api/GVT-g_api.rst

```diff
@@ -0,0 +1,114 @@
+.. _GVT-g_api:
+
+===============
+ACRN GVT-g APIs
+===============
```

Comment:
top level headings after the title use ``*****`` for the underline, so change all your uses of ``=====`` to ``******``